draft-ietf-mls-protocol.md

```diff
@@ -1732,29 +1748,29 @@ A member of the group applies a Remove message by taking the following steps:
 
 * Blank the intermediate nodes along the path from the removed leaf to the root
 
-### External Proposals
+### External Proposals {#external-proposals}
 
 Add and Remove proposals can be constructed and sent to the group by a party
 that is outside the group.  For example, a Delivery Service might propose to
 remove a member of a group has been inactive for a long time, or propose adding
-a newly-hired staff member to a group representing a real-world team. Proposals
-originating outside the group are identified by having a `sender` value in the
-range 0xFFFFFF00 - 0xFFFFFFFF.
-
-The specific value 0xFFFFFFFF is reserved for clients proposing that they
-themselves be added.  Proposals with types other than Add MUST NOT be sent with
-this sender index.  In such cases, the MLSPlaintext MUST be signed with the
-private key corresponding to the ClientInitKey in the Add message.  Recipients
-MUST verify that the MLSPlaintext carrying the Proposal message is validly
-signed with this key.
-
-The remaining values 0xFFFFFF00 - 0xFFFFFFFE are reserved for signer that are
+a newly-hired staff member to a group representing a real-world team.  Proposals
+originating outside the group are identified by an `external` SenderType in
```

Comment:
Note that `external` is not defined above, but probably should be :)

Also, this should be "`external` or `new_member`" (or whatever words we decide on).